lib/ansible/modules/cloud/amazon/cloudfront_distribution.py

```diff
@@ -180,6 +182,7 @@
             I(max_ttl)
             I(compress)
             I(lambda_function_associations[])
+            I(field_level_encryption_id)
```

Comment:
It's not actually required. You can set it if you choose, otherwise we just default it to the empty string (which would ideally be what AWS do, but they don't).

Reply:
Ah, I see. :-)